packages/wallets/src/features/accounts/screens/ResubmitPOA/ResubmitPOA.tsx

```diff
@@ -16,33 +20,102 @@ const ResubmitPOA: React.FC = () => {
+                            label='State/Province'
+                            list={statesList}
+                            listHeight='sm'
+                            name='wallets-poa__address__dropdown-state'
+                            onSelect={handleSelect}
+                            value={selectedState}
+                        />
+                    </div>
+                    <FlowTextField label='Postal/ZIP Code' name='zip-line' />
+                </div>
+            </div>
+
+            <div className='wallets-poa__document'>
+                <div className='wallets-poa__document__title'>
+                    <WalletText weight='bold'>Document Submission</WalletText>
+                    <div className='wallets-poa__document__title__divider' />
+                </div>
+                <div className='wallets-poa__document__container'>
+                    <div className='wallets-poa__document__container__disclaimer'>
+                        <WalletText size='sm' weight='bold'>
+                            We accept only these types of documents as proof of address. The document must be recent
+                            (issued within last 6 months) and include your name and address:
+                        </WalletText>
+
+                        <ul className='wallets-poa__document__container__disclaimer__list'>
+                            <li>
+                                <WalletText size='sm'>
+                                    Utility bill: electricity, water, gas, or landline phone bill.
+                                </WalletText>
+                            </li>
+                            <li>
+                                <WalletText size='sm'>
+                                    Financial, legal, or government document: recent bank statement, affidavit, or
+                                    government-issued letter.
+                                </WalletText>
+                            </li>
+                            <li>
+                                <WalletText size='sm'>Home rental agreement: valid and current agreement.</WalletText>
+                            </li>
+                        </ul>
+                    </div>
+                    <div className='wallets-poa__document__container__upload'>
+                        <WalletText size='sm' weight='bold'>
+                            Upload File
+                        </WalletText>
+                        <Dropzone
+                            description='Remember, selfies, pictures of houses, or non-related images will be rejected.'
+                            descriptionColor='primary'
+                            descriptionSize='sm'
+                            fileFormats={['image/jpeg', 'image/jpg', 'image/png', 'image/gif', 'application/pdf']}
+                            hoverMessage='Upload your file here'
+                            icon={<Upload />}
+                            maxSize={8388608}
```

Comment:
Is the size has limitation of 8mb also for this?

Reply:
Yeap